packages/e2e-tests/specs/autosave.test.js

```diff
@@ -101,6 +102,43 @@ describe( 'autosave', () => {
 		expect( await getEditedPostContent() ).toEqual( wrapParagraph( 'before save after save' ) );
 	} );
 
+	it( 'should clear sessionStorage upon user logout', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'before save' );
+		await saveDraft();
+
+		// Fake local autosave
+		await page.evaluate( ( postId ) => window.sessionStorage.setItem(
```

Comment:
Why not just wait for an actual autosave?

This also makes me think that  using``wp-autosave-block-editor-post-${ postId }`` is too fragile for e2e tests. Ideally these tests should be very high level and emulating a user as much as possible. Details like that should be handled by the relevant API. E.g. here we probably should be using selectors that read from data that syncs with `sessionStorage` and avoid touching it directly.

Reply:
> Why not just wait for an actual autosave?

In short, to save time in E2E execution, since this suite is already long enough. I wouldn't fake it if the previous tests didn't cover it properly. If not for this requirement, I agree with keeping things high-level and emulating the user.

Reply:
Makes sense. But, instead of playing with `sessionStorage` directly, can we at least fire whatever triggers it instead, so that we use the same I/O code and this doesn't depend on the implementation details.

Reply:
> Makes sense. But, instead of playing with `sessionStorage` directly, can we at least fire whatever triggers it instead, so that we use the same I/O code and this doesn't depend on the implementation details.

I agree. However, right now, the writing to `sessionStorage` lives in a callback inside `LocalAutosaveMonitor`, so there is nothing to trigger from the console. However, this will probably change when working on #17501, perhaps by introducing a proper action.

Reply:
That would be better yes, but we can merge this for now.